doc/changelog/pendings/changes_1624436208627288700.md

```diff
@@ -0,0 +1,11 @@
+### Added
+- New cmake option `WITH_STRICT_DATABASE_TYPE_CHECKS` (#5611)
+
+### Changed
+- Prevent mixing databases with query holders (#5611)
+- Prevent using prepared statements on wrong database (#5611)
+- Prevent committing transactions started on a different database (#5611)
+- Convert async queries to new query callbacks (#5611)
+
+### How to upgrade
+- Just see pr (#5611) for exapmle
```

Comment:
Can you add the markdown link here, please? Also, in #5611 I cannot find a how-to-upgrade guide.

Reply:
Added